src/app/@ansyn/ansyn/modules/menu-items/cases/effects/cases.effects.ts

```diff
@@ -124,9 +130,9 @@ export class CasesEffects {
 	@Effect()
 	loadDefaultCase$: Observable<SelectDilutedCaseAction | BackToWorldView> = this.actions$.pipe(
 		ofType(CasesActionTypes.LOAD_DEFAULT_CASE),
-		withLatestFrom(this.store.select(selectActiveMapId)),
-		filter(([action, mapId]: [LoadDefaultCaseAction, string]) => !action.payload.context),
-		mergeMap(([action, mapId]: [LoadDefaultCaseAction, string]) => {
+		withLatestFrom(this.store.select(selectMapsIds)),
```

Comment:
don't you should iterate over all the maps id?

Reply:
no, because i only need the default map, which is the first map in the array